iset.mm

```diff
@@ -126196,6 +126239,15 @@ class of all (base sets of) groups is proper.  (Contributed by Mario
     ndxid $p |- E = Slot ( E ` ndx ) $=
       ( cnx cfv wceq cslot ndxarg eqcomi sloteq syl5eq ax-mp ) BEAFZGZANHZGNBAB
       CDIJOABHPCBNKLM $.
+
+    $( A structure component extractor is defined by its own index.  That the
+       index is a natural number will also be needed in quite a few contexts so
+       it is included in the conclusion of this theorem which can be used as a
+       hypothesis of theorems like ~ strslfv .  (Contributed by Jim Kingdon,
+       29-Jan-2023.) $)
+    ndxslid $p |- ( E = Slot ( E ` ndx ) /\ ( E ` ndx ) e. NN ) $=
```

Comment:
Would it be useful in set.mm, too?

Reply:
You've done a lot of the recent work in set.mm so you might have as many thoughts on that as I, but I would think that it could be.  Presumably it would be useful if done in conjunction with rearranging some of the theorems which would use it, that is those that have separate ``E = Slot ( E ` ndx )`` and ``( E ` ndx ) e. NN`` hypotheses (or variations on that which amount to the same thing).